src/Leela.cpp

```diff
@@ -118,6 +118,7 @@ static void parse_commandline(int argc, char *argv[]) {
         ("puct", po::value<float>())
         ("softmax_temp", po::value<float>())
         ("fpu_reduction", po::value<float>())
+        ("fpu_root_reduction", po::value<float>())
```

Comment:
If this option is to be useful for analysis, can you please explain why it is a "tuning option"?